### DISCUSSION TITLE
[feat request]: 允许只翻译 不还原

### DISCUSSION THREAD

**Discussion by zhangenming** [immersive-translate/immersive-translate#2207] [Ideas]
### 新功能 | New Feature

![image](https://github.com/user-attachments/assets/4cb69dc9-7bad-4d8f-9e49-fe0bee36dbc7)
连续俺两次alt会导致隐藏翻译结果
希望可以提供选项
是否打开 第二次按键后还原翻译结果

### 为什么您想要添加这些功能 | Why you want to add these features

1.
2.
3.

**Comment by theowenyoung** [comment]
你需要按 一次 alt 就好了呀，为啥要连续按2次呢？

**Reply by zhangenming** [reply]
因为我另有其他目的 这个就和它冲突了
